examples/go-client/main.go

```diff
@@ -18,26 +18,70 @@ package main
 
 import (
 	"context"
-	"fmt"
 	"log"
+
+	"github.com/gravitational/teleport/api/client"
+	"github.com/gravitational/teleport/api/types"
+
+	"github.com/gravitational/trace"
+	"github.com/pborman/uuid"
 )
 
 func main() {
+	ctx := context.Background()
 	log.Printf("Starting Teleport client...")
-	client, err := connectClient()
+
+	var creds *client.Credentials
+	var err error
+
+	clt, err := client.New(client.Config{
+		// Addrs is the Auth Server address, only works locally.
+		// TODO (Joerger): add support to connect via proxy.
```

Comment:
Let's not add TODO's here, because I think this gets pulled into our documentation right?